[2023-10-06 14:05:05,154] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.retrieve_raw_data 2023-10-06T14:03:48.810481+00:00 [queued]>
[2023-10-06 14:05:05,167] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.retrieve_raw_data 2023-10-06T14:03:48.810481+00:00 [queued]>
[2023-10-06 14:05:05,168] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2023-10-06 14:05:05,168] {taskinstance.py:1088} INFO - Starting attempt 1 of 6
[2023-10-06 14:05:05,168] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2023-10-06 14:05:05,184] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): retrieve_raw_data> on 2023-10-06T14:03:48.810481+00:00
[2023-10-06 14:05:05,192] {standard_task_runner.py:52} INFO - Started process 27165 to run task
[2023-10-06 14:05:05,194] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', '***_exam', 'retrieve_raw_data', '2023-10-06T14:03:48.810481+00:00', '--job-id', '265', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmplublwfeb', '--error-file', '/tmp/tmpb0msvew1']
[2023-10-06 14:05:05,195] {standard_task_runner.py:77} INFO - Job 265: Subtask retrieve_raw_data
[2023-10-06 14:05:05,234] {logging_mixin.py:104} INFO - Running <TaskInstance: ***_exam.retrieve_raw_data 2023-10-06T14:03:48.810481+00:00 [running]> on host 3a3d0401ac26
[2023-10-06 14:05:05,291] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=***_exam
AIRFLOW_CTX_TASK_ID=retrieve_raw_data
AIRFLOW_CTX_EXECUTION_DATE=2023-10-06T14:03:48.810481+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-10-06T14:03:48.810481+00:00
[2023-10-06 14:05:05,687] {python.py:151} INFO - Done. Returned value was: None
[2023-10-06 14:05:05,698] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=***_exam, task_id=retrieve_raw_data, execution_date=20231006T140348, start_date=20231006T140505, end_date=20231006T140505
[2023-10-06 14:05:05,734] {taskinstance.py:1265} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2023-10-06 14:05:05,769] {local_task_job.py:151} INFO - Task exited with return code 0
